INSTALLATION.md

```
@@ -0,0 +1,337 @@
+      ```bash
+      $ git clone https://github.com/PalisadoesFoundation/talawa-admin.git
+      ```
+
+## Install node.js
+
+Best way to install and manage `node.js` is making use of node version managers. We recommend using `fnm`, which will be described in more detail later.
+
+Follow these steps to install the `node.js` packages in Windows, Linux and MacOS.
+
+1. For Windows:
+   1. first install `node.js` from their website at https://nodejs.org
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)